code/web/src/modules/user/api/actions.test.js

```diff
@@ -0,0 +1,27 @@
+import * as userAction from "./actions";
+
+describe('User Actions', () => {
```

Comment:
Thanks for getting us started on testing these actions.  This is pretty much the way they taught us to test Redux actions back when they were still testing it.